apis/management.cattle.io/v3/cluster_scan_types.go

```diff
@@ -104,3 +130,22 @@ type CisBenchmarkVersion struct {
 
 	Info CisBenchmarkVersionInfo `json:"info" yaml:"info"`
 }
+
+type ScheduledClusterScanConfig struct {
+	// Cron Expression for Schedule
+	CronSchedule string `yaml:"cron_schedule" json:"cronSchedule,omitempty"`
+	// Number of past scans to keep
+	Retention int `yaml:"retention" json:"retention,omitempty"`
+}
+
+type ScheduledClusterScan struct {
+	// Enable or disable scheduled scans
+	Enabled        bool                        `yaml:"enabled" json:"enabled,omitempty" norman:"default=false"`
+	ScheduleConfig *ScheduledClusterScanConfig `yaml:"schedule_config" json:"scheduleConfig,omitempty"`
```

Comment:
this is a bit confusing, since we have both schedule and scan prefixed `ClusterScanConfig`...one's a scheduleConfig and another's a clusterScanConfig

Reply:
can we have struct names also similar to yaml and json tags? they look correct

Reply:
I used `schedule_config` to make it easier for user to configure things in cluster.yml, but if we use the same for struct names, v3.ScheduleConfig doesn't make much sense. Hence, I ended up with the current state.